### PR TITLE
msgpack: remove -Werror from subdirectories

### PR DESCRIPTION
**Pull request by chrboe**

### CODE REVIEW

File: dev-libs/msgpack/msgpack-1.2.0.ebuild [hunk 1]
```diff
@@ -31,6 +31,12 @@ DOCS=( README.md )
 
 src_prepare() {
 	sed -i -e 's,-Werror,,g' CMakeLists.txt || die
+	sed -i -e 's,-Werror,,g' CMakeLists.txt || die
```
Comment (l29ah) [inline dev-libs/msgpack/msgpack-1.2.0.ebuild]: an extra one? also maybe you should've wildcarded the dirs

Comment (chrboe) [reply dev-libs/msgpack/msgpack-1.2.0.ebuild]: whoops, dumb copy and paste error.

will `**/*/CMakeLists.txt` work or do we need some kind of `find` magic?

Comment (l29ah) [reply dev-libs/msgpack/msgpack-1.2.0.ebuild]: it should work

**Comment by l29ah** [comment]
You should've squashed your commits, but okay.

-- 
()  ascii ribbon campaign - against html mail
/\  http://arc.pasp.de/   - against proprietary attachments

**Comment by chrboe** [comment]
fyi, there's a squash option when merging a PR: https://help.github.com/articles/about-pull-request-merges/

you're welcome

**Pull request merged**